new/LIT6845PPSatolay.xml

```diff
@@ -0,0 +1,75 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against <foreign xml:lang="gez">šatolāy</foreign>, with <foreign xml:lang="gez">ʾasmāt</foreign></title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2023-06-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6845PPSatolay" passive="BDLaethg47"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The following is the entire text of the prayer, taken from <ref type="mss" corresp="BDLaethg47"/>.</note>
+                    <ab>
+                        በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩
+                        አምላክ<supplied reason="omitted">፡</supplied> ጸሎት፡ በእንተ፡ ሕማመ<supplied reason="omitted">፡</supplied>
+                        ሸቶላይ፡ አንተ፡ ሸቶላይ፡ ወሸቶላዊ፡ ጋኔን፡ ዘ<sic>ትቀት<supplied reason="omitted">፡</supplied></sic> ሕፃናተ፡ ወትቋርጽ፡ አማሉተ፡ ወትጦዊ፡ አንጀተ፡
+                        ወትቆረጥም፡ አካላተ፡ ወተሐምም፡ አባላተ፡ አምሐልኩከ፡ ወአውገዝኩከ፡ ወለጐምኩከ፡ ወአስርኩከ፡ ወቆለፍኩከ፡ ወሐተምኩከ፡ በማሕተመ<supplied reason="omitted">፡</supplied>
```

Comment:
It should be አሰርኩከ (from አሰረ, https://betamasaheft.eu/Dillmann/?mode=none&q=%E1%8A%A0%E1%88%B0%E1%88%A8&id=L0254f16a079a407bb7829d29f199f4f9). I would correct this or nest the word inside `<sic/>` (also in the MS record)

Reply:
Thank you, also corrected in the manuscript